src/rpc/mempool.h

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) 2024 Chaintope Inc.
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#ifndef TAPYRUS_RPC_MEMPOOL_H
+#define TAPYRUS_RPC_MEMPOOL_H
+
+class CTxMemPool;
+class UniValue;
+
+/** Mempool information to JSON */
+UniValue MempoolInfoToJSON(const CTxMemPool& pool);
+
+/** Mempool to JSON */
+UniValue MempoolToJSON(const CTxMemPool& pool, bool verbose = false, bool include_mempool_sequence = false);
+
```

Comment:
Where is these implementations?

Reply:
it was incorrectly ported from bitcoin. Its not implemented at all.